packages/client/mutations/UpdateReflectionGroupTitleMutation.ts

```diff
@@ -37,7 +37,7 @@ const UpdateReflectionGroupTitleMutation: StandardMutation<TUpdateReflectionGrou
     onError,
     optimisticUpdater: (store) => {
       const {reflectionGroupId, title} = variables
-      const reflectionGroupProxy = store.get(reflectionGroupId)
+      const reflectionGroupProxy = store.get(reflectionGroupId)!
```

Comment:
in many places I opted to keep the runtime behavior the same because this PR affects so many parts